src/main/java/uk/gov/companieshouse/ordernotification/emailsendmodel/OrderNotificationEmailData.java

```diff
@@ -110,6 +126,22 @@ public void setDispatchDays(int dispatchDays) {
         this.dispatchDays = dispatchDays;
     }
 
+    public boolean isHasStandardDelivery() {
```

Comment:
getters and setters already exist for hasStandardDelivery and hasExpressDelivery